src/main/java/com/afornalik/ox/board/Board.java

```diff
@@ -0,0 +1,35 @@
+package com.afornalik.ox.board;
+
+import java.util.Map;
+
+public class Board {
+
```

Comment:
You can add max value to prevent of making number of fields = Integer.MAX_VALUE * Integer.MAX_VALUE

Reply:
done - created field size

Reply:
size is BigInteger type to hold every number